Makefile

```diff
@@ -110,11 +122,45 @@ clear-cache-prod: install production.ini
 c2corg_ui/closure/%.py: $(CLOSURE_LIBRARY_PATH)/closure/bin/build/%.py
 	cp $< $@
 
-c2corg_ui/locale/c2corg_ui-client.pot: $(APP_HTML_FILES) $(APP_PARTIAL_FILES) $(APP_JS_FILES)
+# i18n and Transifex tools
+
+# if .transifexrc does not exist yet, create it for read only access (with fake user c2c)
+$(HOME)/.transifexrc:
+	echo "[https://www.transifex.com]" > $@
+	echo "hostname = https://www.transifex.com" >> $@
+	echo "username = c2c" >> $@
+	echo "password = c2cc2c" >> $@
```

Comment:
Yes,
This is used to download the .po files, It's required because Transifex requires a user to use the command, and this user has no rights.
The user used to upload the pot is defined here: 
https://github.com/c2corg/v6_ui/pull/217/files#diff-354f30a63fb0907d4ad57269548329e3R30